tests/src/test/scala/ironrust/FullIntegrationTest.scala

```diff
@@ -294,12 +303,8 @@ class FullIntegrationTest extends DudeSuite with CancelAfterFailure {
       val sdk = IronSdk.initialize(createDeviceContext)
       val groupId = Try(GroupId.validate("not-a-group=ID-that-exists=")).toEither.value
       val resp = Try(sdk.groupGetMetadata(groupId)).toEither
-      resp.leftValue.getMessage should include(
-        """404"""
-      )
-      resp.leftValue.getMessage should include(
-        """Requested resource was not found"""
-      )
+      resp.leftValue.getMessage should include("""404""")
```

Comment:
Nit: You don't need `"""`, just normal `"`

Reply:
I didn't put them there originally, but I now took them out